packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.78.3",
+  "version": "3.78.4",
```

Comment:
i think this should be a new minor version since this PR adds a new feature.. not a bugfix

Reply:
done